### DISCUSSION TITLE
2nd Vehicle shows ??? and null

### DISCUSSION THREAD

**Discussion by Ken-Yn** [teslamate-org/teslamate#2696] [Q&A]
Good morning,

I have Tesla mate for a little over 1 year now.  I recently purchased a 2nd Tesla about 5 days ago.  The first vehicle shows the last six of the VIN in both TeslaMate and Graphana.  The new vehicle shows ??? in Teslamate and null in graphana though all data from the vehicle is collected and displayed properly otherwise.

Any idea what is causing this?  Thanks in advance.
<img width="1028" alt="Screenshot 2022-06-10 105844" src="https://user-images.githubusercontent.com/107265197/173093595-bdefb9f0-531a-45ee-a1f9-28fe12dbc816.png">
<img width="1873" alt="Screenshot 2022-06-10 105709" src="https://user-images.githubusercontent.com/107265197/173093596-313a80a6-8ecd-4a0a-aba2-900713be75ba.png">

**Comment by cwanja** [comment]
Have you tried logging out and logging back in? Can you run the following command from the database? `select model, vin from cars;` - you can remove the first handful of characters from the returned VIN for privacy if you would like.

**Reply by Ken-Yn** [reply]
I tried sign out and sign back in but still the same on both.  Stupid question I know but what is procedure for running commands on db?  This is the basic docker install.

**Reply by cwanja** [reply]
No stupid question at all.
1. Go to the directory where you have the YML file
2. Run `docker-compose exec database psql teslamate teslamate`. `database` might be `db` if you get an error. Check your YML file for the "service" and if you have a custom username and or password (other than `teslamate teslamate`
3. Once you are dropped into the database, run the command above.

**Reply by cwanja** [reply]
Would also help to have the Docker logs for five days ago when the vehicle was added, if they are still available.

**Reply by Ken-Yn** [reply]
What is the easiest way to pull the logs?  Looks like the full VIN for the new one is in the db
<img width="495" alt="Screenshot 2022-06-10 114510" src="https://user-images.githubusercontent.com/107265197/173103055-8bed3ba3-0b32-4dd7-9055-d99936e29bb3.png">

**Reply by cwanja** [reply]
No need to pull the logs, that looks like a bug then in the application interface not puling the VIN. I _imagine_ it is because the beginning structure of the VIN changed with the manufacture. Does your other VIN start with 7SA as well? Or was it 5YJ? 

Either way, I think this is a web interface bug. I would covert this discussion into an issue - type "bug" using the link on the right hand side:
![Screen Shot 2022-06-10 at 10 50 35 AM](https://user-images.githubusercontent.com/54645561/173103662-26722388-44fc-455d-9f0f-d4b6428b9400.png)

**Reply by Ken-Yn** [reply]
Yes first vehicle starts with 5YJ .

**Reply by cwanja** [reply]
Yep, I believe the web interface does not know how to handle a VIN with 7SA as the manufacture code in the VIN. Create an issue bug for this :) 

@adriankumpf  FYI.
